config/locales/de.yml

```diff
@@ -217,23 +373,191 @@ de:
+          ein Ereignis bewerten können.
+        max_timeslots: Was ist die maximale Länge einer Einreichung (gerechnet in Zeit-Slots)?
+        timeslot_duration: |
+          Wie lange sollen die kleinsten Zeit-Slots dauern? Hierdurch wird das grundlegende Zeit-Raster
+          der Veranstaltungs aufgespannt, mit dem Vorträge und Pausen geplant werden können. WARNUNG:
+          der Wert kann später nicht mehr so leicht geändert werden und ist für Ungeübte schwer abzusehen.
+          Als Tip: Im Allgemeinen machen größere Zeitfenster die Planung einfacher, aber kleinere flexibler.
+          Beispiel: Wenn Sie nur mit halbstündigen Vorträgen planen, die auch genau zu vollen oder halben
+          Stunden beginnen, wählen Sie 30 Minuten als Zeit-Slot. Wenn Sie nur mit ganzstündigen Vorträge
+          planen und dazwischen 15 Minuten Pausen, ist 15 Minuten eine gute Wahl.
+          Beachten Sie, dass Sie die Größe des Slots später verkleinern können, solange die
+          alte Slot-Größe ein Vielfaches der neuen ist. Also 15-Minuten-Slots auf 5 zu verringern funktioniert,
+          während eine Änderung auf 10 Minuten nicht möglich ist.
+        timezone: |
+          Bitte wählen Sie aus, in welcher Zeitzone Ihre Konferenz stattfinden soll.
+          Dies ist erforderlich, um die Zeiten für Ihre Events korrekt anzuzeigen.
+        title: |
+          Der Name der Konferenz, wie sie auf der gesamten Website erscheinen soll.
+          Beispiel: 'FrOSCon'
+        transport_needs_enabled: |
+          Wenn aktiviert, können Sie mit frab den Transportbedarf der Sprecher verwalten
+    manage_conferences: Konferenzen verwalten
+    manage_conferences_hint: Verwalten Sie alle Konferenzen
+    new: Neu
+    new_welcome: |
+      Herzlich willkommen! Es sieht so aus, als wären Sie zum ersten Mal hier.
+      Um mit der Konferenz-Organisation zu beginnen, müssen Sie zuerst einige Daten angeben.
+      Sie können das meiste (mit Ausnahmen) später noch ändern. Daher lesen Sie die Hinweise
+      bitte sorgfältig durch.
+    no_conference: Es gibt noch keine Konferenzen.
+    no_conference_with_cfp: Es gibt keine aktuelle Konferenz mit einem Call for Participation. Vielleicht müssen Sie sich zuerst einloggen?
+    no_conferences: |
+      Es gibt noch keine Konferenzen.
+      Fügen Sie mit der Schaltfläche rechts eine Konferenz hinzu.
+    notice_bulk_notification_queued: Alle Benachrichtigungen für die Einreichungen in %{notification} an den Ticket-Server übergeben.
+    notice_conference_created: Die Konferenz wurde erfolgreich erstellt.
+    notice_conference_updated: Die Konferenz wurde erfolgreich aktualisiert.
+    past_conferences: Frühere Konferenzen
+    recent_changes: Letzte Änderungen
+    recent_changes_text: |
+      Wenn Sie oder andere Änderungen an der Konferenz oder Einreichungen vornehmen, können Sie
+      diese hier zusammengefasst überblicken.
+    rejected: rejected
+    rejecting: rejecting
+    review: review
```

Comment:
Hier kenne ich leider den Kontext nicht.
Entweder "bewerten" (im Sinne von 1-5 Sterne) oder "prüfen" (im Sinne von Daten noch einmal überprüfen).

Reply:
Please see my comment on this issue, where I questioned the use of reviewing as a german verb, because the "review" term is also used later in the file.